### DISCUSSION TITLE
Step-by-Step: How to Setup Copilot Chat in VS Code

### DISCUSSION THREAD

**Discussion by WirelessLife** [community/community#58756] [Copilot Conversations] [answered]
<img width="1479" alt="Screenshot 2023-06-29 at 11 57 16 AM" src="https://github.com/community/community/assets/95834130/43b76070-4c67-42bf-bd80-961f50d4fd20">

A quick run through of setting up Copilot Chat on VS Code once you've received confirmation of access to the preview. Happy to address any questions anyone may have. Read more here: [Step-by-Step: How to Setup Copilot Chat in VS Code](https://techcommunity.microsoft.com/t5/educator-developer-blog/step-by-step-how-to-setup-copilot-chat-in-vs-code/ba-p/3849227?WT.mc_id=academic-99253-abartolo)

Reynald Adolphe also has a [great video walkthrough](https://youtu.be/eCOsNCYxdYM) of the GitHub Copilot chat setup process which can be viewed below.

**Comment by 6DiegoDiego9** [comment]
![image](https://github.com/community/community/assets/109476941/1e7d487c-c9a4-4cbe-bad5-ef43269331b2)

I reinstalled Copilot Chat and restarted VS Code 10 times and I still can't see its icon on the left :-(

**Reply by WirelessLife** [reply]
Did you verify your access to Copilot Chat was granted? Here is the link to verify: https://github.com/github-copilot/chat_waitlist_signup/join

**Comment by 6DiegoDiego9** [comment]
Ah, no, I apparently was not in that waitlist. I read nowhere that I needed to do that.
I joined now. I hope to having to wait very much.
Thanks for the info!

**Comment by nemavixd** [comment]
I get this when I open Visual studio code 
![Screenshot 2024-02-10 193854](https://github.com/community/community/assets/159462880/5923ee53-f215-4914-aa10-d9e835129dfd)

**Reply by nemavixd** [reply]
I created an account recently on github

**Reply by WirelessLife** [reply, marked as answer]
Click on the "Signup for GitHub Copilot" button to enable the service.  This provides a time limited free trial of GitHub Copilot.

**Reply by nemavixd** [reply]
already did, still the same
